endpoint_logger_response.py

```
@@ -0,0 +1,81 @@
+from mitmproxy import http
+from datetime import datetime
+import json
+import os
+import time
+import base64
+
+class EndpointLogger:
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)